packages/api/endpoints/granules.js

```diff
@@ -114,17 +113,15 @@ const create = async (req, res) => {
       dynamoRecord: granule,
       knexOrTransaction: knex,
     });
-    if (
-      await granulePgModel.exists(knex, {
-        granule_id: pgGranule.granule_id,
-        collection_cumulus_id: await collectionPgModel.getRecordCumulusId(
-          knex,
-          deconstructCollectionId(granule.collectionId)
-        ),
-      })
-    ) {
+
+    // TODO: CUMULUS-3017 - Remove this unique collectionId condition
+    //  and only check for granule existence
```

Comment:
Are these comments helpful for when we implement https://bugs.earthdata.nasa.gov/browse/CUMULUS-3017 ?